cumulus/common/girder.py

```diff
@@ -28,6 +28,8 @@
 import cumulus
 from cumulus.constants import ClusterType
 
+from cumulus_plugin.models.aws import Aws as AwsModel
```

Comment:
@cjh1 I added imports at the tops of the files for the plugin models that were needed. Then later (below), I just use the model directly.

This uses less code than `ModelImporter` (see below for an example), because it skips the registration. But I can still use the model importer if needed.

```But
from cumulus_plugin.models.aws import Aws as AwsModel
ModelImporter.registerModel('aws', AwsModel, 'cumulus')
...
profile = ModelImporter.model('aws', 'cumulus').findOne(query)
```